ngrinder-frontend/package.json

```diff
@@ -22,8 +22,8 @@
     "axios": "^0.21.1",
     "base64-js": "^1.2.1",
     "billboard.js": "^1.11.0",
-    "bootbox": "^5.2.0",
-    "bootstrap": "4.3.1",
+    "bootbox": "^5.5.3",
```

Comment:
Many of npm dependencies we are using have been deprecated. I didn't fix it in this PR, but it needs to be fixed soon.